packages/message-input/src/vaadin-message-input-mixin.js

```diff
@@ -111,8 +116,17 @@ export const MessageInputMixin = (superClass) =>
             }
           });
 
-          const input = textarea.inputElement;
-          input.removeAttribute('aria-labelledby');
+          let input = textarea.inputElement;
+
+          // With Lit version, input element renders asynchronously and it will
+          // override the `rows` attribute set to `1` in the `minRows` observer.
+          // Workaround by creating a custom textarea instead to not update it.
+          // TODO: needs https://github.com/vaadin/web-components/pull/8168
```

Comment:
Note: we don't have snapshot tests for the Lit version, maybe we should have them as unit tests don't cover this.